keyboards/signum31/config.h

```diff
@@ -0,0 +1,44 @@
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xF1E7
+#define PRODUCT_ID      0x5163
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    troyfletcher
+#define PRODUCT         keyboard
+#define DESCRIPTION     Keyboard
+
+/* key matrix size */
+#define MATRIX_ROWS 6
+#define MATRIX_COLS 10
+
+/* key matrix pins */
+#define MATRIX_ROW_PINS { D2, D1, F5, B5, B2, C6 }
+#define MATRIX_COL_PINS { B4, D7, D0, E6, D4, F6, F4, F7, B1, B3 }
+#define UNUSED_PINS
+
+/* COL2ROW or ROW2COL */
+#define DIODE_DIRECTION COL2ROW
+
+/* number of backlight levels */
+
+#ifdef BACKLIGHT_PIN
+#define BACKLIGHT_LEVELS 0
+#endif
+
+/* Set 0 if debouncing isn't needed */
+#define DEBOUNCING_DELAY 5
```

Comment:
This setting was renamed.

```suggestion
#define DEBOUNCE 5
```